app/view_models/kba_security_show.rb

```diff
@@ -0,0 +1,16 @@
+class KbaSecurityShow
+  AGENCIES = [
+    ['CBP Trusted Traveler Programs', 1],
+    ['USA JOBS', 2],
+    ['Railroad Retirement Board', 3],
+    ['NSG Open Mapping Enclave', 4],
+  ].freeze
+
+  def self.answers
+    arr = []
+    arr << [I18n.t('kba_security.dropdown_message'), -1]
+    arr.concat(AGENCIES)
+    arr << [I18n.t('kba_security.dropdown_other'), 0]
+    arr
+  end
+end
```

Comment:
I agree with Eric that this should be tested with users and grounded in research. I would be surprised if legitimate users who have visited more than one agency would always know which one they've last visited.

Did a meeting already take place to discuss additional options?

Reply:
Also, this gives malicious users a 1/4 chance of guessing the correct answer.

Reply:
I mentioned we could offer a timeframe option as a second KBA question (last week, last month, last year, etc.).  The KBA is in addition to the security we already have which is getting notified by phone and email that you are changing the phone. You have the option to cancel and/or report the event for 72 hours. Then you get a UUID/token that's good for 24 hours and you still have to login with username/password to change the phone.  We could even tighten up how long the token is good for.